src/handler/get.ts

```diff
@@ -1,24 +1,44 @@
+import { GetParameterCommand, SSMClient } from '@aws-sdk/client-ssm';
+import type { APIGatewayProxyResult } from 'aws-lambda';
 import 'dotenv/config';
-import type { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
 import logger from '../util/logger';
 
-// eslint-disable-next-line @typescript-eslint/require-await
-export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
-  if (!event.queryStringParameters) {
-    return {
-      statusCode: 400,
-      body: JSON.stringify({
-        message: 'Missing query string parameters',
-      }),
-    };
-  }
+const headers = {
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': 'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token',
+  'Access-Control-Allow-Methods': 'DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT',
```

Comment:
If this is a GET request, do we need the other HTTP verbs listed here?

Reply:
Ok np, I can understand why this might be needed if the resource was using a `/{proxy+}` for the route - but worth a check nonetheless.

Reply:
works now 🤷 nice one haha